packages/client/src/components/Visual.test.tsx

```diff
@@ -31,4 +31,38 @@ describe("Visual", () => {
     expect(screen.queryByText("title")).toBeInTheDocument();
     expect(screen.queryByTestId(FIGCAPTION)).toBeInTheDocument();
   });
+
+  it("renders correctly with on error", () => {
+    const { container } = render(
+      <Visual alt="alt text" src="folder/image.jpg" title="title" />
+    );
+
+    expect(screen.queryByAltText("alt text")).toBeInTheDocument();
+    expect(screen.queryByText("title")).toBeInTheDocument();
+    expect(screen.queryByTestId(FIGCAPTION)).toBeInTheDocument();
```

Comment:
All duplicate `expect` lines can be removed

Reply:
Yes nice!